spec/lib/process_settings/target_spec.rb

```diff
@@ -134,6 +134,54 @@
       expect(process_target.target_key_matches?(context_hash)).to be_truthy
       expect(process_target.target_key_matches?({})).to be_falsey
     end
+
+    context "for substring matching" do
+      it "should match on values when using backslash delimiters in target hash" do
```

Comment:
You can DRY up the tests below by putting some common `let`s up here:
```
let(:context_hash) do
  {
    'service' => service,
    'region' => 'east',
    'cdr' => { 'caller' => '+18056807000' }
  }
end
```
The individual contexts just need to do `let(:service) { ... }`.